pkg/graphql/schema.go

```diff
@@ -134,3 +135,9 @@ func createSchema(schemaContent []byte) (*Schema, error) {
 		document: document,
 	}, nil
 }
+
+func SchemaIntrospection(schema *Schema) (*ExecutionResult, error) {
+	var buf bytes.Buffer
+	err := schema.IntrospectionResponse(&buf)
```

Comment:
Why not just `schema.IntrospectionResponse` returns `(*ExecutionResult, error)`?

Reply:
to have more control over how you want to get a response as an ExecutionResult or as a plain JSON